### PR TITLE
Update upstream source links

### PR DESCRIPTION
**Pull request by kasicka**
Fix some typos

### CODE REVIEW

File: src/components/PageHelp.tsx [hunk 1]
```diff
@@ -237,23 +237,37 @@ const Help = () => (
...
-        <p>
-            {"Dashboard's"} backend - Kai - is developed separately. You can
-            find the code here:
-        </p>
-        <p className="padding-left-20">
-            <a href="https://gitlab.cee.redhat.com/baseos-qe/kai">
-                https://gitlab.cee.redhat.com/baseos-qe/kai
-            </a>
-        </p>
+        <table style={{ width: '45%' }}>
```
Comment (mgrabovsky) [inline src/components/PageHelp.tsx:243]: I'm not a big fan of tables for this use case. I'd prefer to use a [list](https://www.patternfly.org/v4/components/list/) here. You can also use the [`ExternalLink` component](https://github.com/fedora-ci/ciboard/blob/2770c41f5cb3718f50a60bb57e150f9b2d8e93b6/src/components/ExternalLink.tsx) for the links.

Comment (kasicka) [reply src/components/PageHelp.tsx:243]: I would've used list, but I found this kind of alignment nicer visually.

**Review by mgrabovsky — APPROVED** [review]
Looks good to me.

**Comment by mgrabovsky** [comment]
@Andrei-Stepanov Are you OK with these changes. They're pretty much just link fixes.

**Review by Andrei-Stepanov — APPROVED** [review]

**Comment by Andrei-Stepanov** [comment]
Hi, please merge.

@mgrabovsky  I do not remember. If there is something it was left at:

https://gitlab.cee.redhat.com/osci/ciboard
or even previous re-incarnation:
https://gitlab.cee.redhat.com/osci/ci-dashboard

**Comment by mgrabovsky** [comment]
The pictures are in osci/ci-dashboard, but they're quite outdated, so it might be best to remove them.

**Pull request merged**